tools/clang/test/DXC/extra_output.test

```diff
@@ -0,0 +1,9 @@
+// Test extra DXC outputs together
+// RUN: %dxc /T ps_6_0 %S/Inputs/smoke.hlsl /DDX12 /Dcheck_warning /Fh %t.h /Vn g_myvar /Fc %t.ll /Fo %t.cso /Fre %t.reflection /Frs %t.rootsig /Fe %t.err
+// RUN: FileCheck --input-file=%t.h %s --check-prefix=HEADER | FileCheck --input-file=%t.ll %s --check-prefix=IR
```

Comment:
The `|` gets interpreted as a shell pipe. Probably better to just put the second FileCheck command on a separate line since they are both reading from a file.

Reply:
The test will fail for cannot find the temp file generated by first line if use '|'.

Reply:
Oh, that probably has to do with when lit decides the temporary should be cleaned up. So maybe instead use `&&`.

Reply:
You didn’t actually fix this…

I think we also need to understand why this causes an issue if the FileCheck lines are separate.

Reply:
Fixed.
I don't think it is a problem since upstream use FileCheck in separate RUN lines too.
Maybe we could fix it in upstream first then merge back?